packages/portalnetwork/src/networks/network.ts

```diff
@@ -328,16 +329,27 @@ export abstract class BaseNetwork extends EventEmitter {
               `Client ${shortId(enr.nodeId)} is ${decodeClientInfo(ClientInfo).clientName} node with capabilities: ${Capabilities}`,
             )
             this.routingTable.updateRadius(enr.nodeId, DataRadius)
+            this.portal.enrCache.updateNodeFromPong(enr, this.networkId, {
+              capabilities: Capabilities,
+              clientInfo: decodeClientInfo(ClientInfo),
+              radius: DataRadius,
+            })
             break
           }
           case PingPongPayloadExtensions.BASIC_RADIUS_PAYLOAD: {
             const { dataRadius } = BasicRadius.deserialize(pongMessage.customPayload)
             this.routingTable.updateRadius(enr.nodeId, dataRadius)
+            this.portal.enrCache.updateNodeFromPong(enr, this.networkId, {
+              radius: dataRadius,
+            })
             break
           }
           case PingPongPayloadExtensions.HISTORY_RADIUS_PAYLOAD: {
             const { dataRadius } = HistoryRadius.deserialize(pongMessage.customPayload)
             this.routingTable.updateRadius(enr.nodeId, dataRadius)
+            this.portal.enrCache.updateNodeFromPong(enr, this.networkId, {
+              radius: dataRadius,
+            })
             break
           }
           case PingPongPayloadExtensions.ERROR_RESPONSE: {
```

Comment:
I can't comment below but should we do some peer scoring on calls to `routingTable.evictNode`?

Reply:
Yes.  

Currently we don't have any of the peer scoring implemented.  I figured that would come next, after getting this in place.  Not sure which metrics are really worth tracking, or when to actually look at a peer's score, and what decisions to make based on a peer score.  

I do believe that some sort of scoring and peer metrics are helpful, but I think the details of that warrant some discussion.